### PR TITLE
Added mark.js

### PR DESCRIPTION
**Pull request by julmot**
Btw: I'd suggest to implement mark.js as a keyword highlighter for the [search](http://www.jsdelivr.com/?query=tes).

### CODE REVIEW

**Comment by julmot** [comment]
Ping @jimaek

File: files/mark.js/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "bower",
+  "name": "mark.js",
+  "repo": "julmot/mark.js",
+  "files": {
+    "basePath": "dist/"
```
Comment (jimaek) [inline files/mark.js/update.json]: How about adding "include" with `*.min.*` to avoid the unminified files?

Comment (julmot) [inline files/mark.js/update.json]: Why don't let developers choose if they want to use the minified one or not?

Comment (julmot) [inline files/mark.js/update.json]: Ping @jimaek

Comment (jimaek) [inline files/mark.js/update.json]: jsDelivr is focused on production usage. But if you need the unminified files then we can leave them.

Comment (julmot) [inline files/mark.js/update.json]: Please let them for consistency as I'm offering them in all other distribution processes.

**Comment by julmot** [comment]
@jimaek Thank you for merging! 

Seems like the current offered versions are quite incomplete. Does this happen due to the fact that the bot is still busy autoupdating or is there any manual help necessary?

**Comment by jimaek** [comment]
We only host the latest version from each major branch. Like I said its production focused.
So the bot wont upload any older versions anymore. If you really require some older versions for some reason then you will have to add them manually

**Comment by julmot** [comment]
@jimaek Is this really true? I've seen a few different projects offering all versions, e.g.
https://github.com/jsdelivr/jsdelivr/tree/master/files/markdown-it

What I'd like to have is a consistent behavior. I'm offering all versions in every distribution process, also e.g. on cdnjs: https://cdnjs.com/libraries/mark.js 
I'd like to offer jsdelivr as the preferred CDN, but can't do that if there aren't all files and all versions. 

If I understand you correctly the bot will delete previous major versions when there is a new one?

**Comment by megawac** [comment]
@julmot while we will continue versions we already host for all time, we won't automatically fetch old versions

**Comment by jimaek** [comment]
markdown-it has all the versions because auto-updates were enabled a long time ago and the bot simply added new releases over time. it didnt upload everything in 1 PR.

The bot will NOT delete anything. It will simply continue adding new versions.

**Comment by julmot** [comment]
Alright, that said I'll have to add all previous versions manually but the bot will keep adding all new versions regardless if they are major, minor or patch?

**Comment by jimaek** [comment]
Thats correct.

**Pull request merged**